gateway/test/chains/avalanche/avalanche.routes.test.ts

```diff
@@ -20,7 +20,7 @@ afterEach(unpatch);
 
 describe('GET /avalanche', () => {
   it('should return 200', async () => {
-    request(app)
+    await request(app)
```

Comment:
I don't think you need the await here (it's not wrong, but maybe unnecessary), the test itself is a promise chain and it won't be resolved until the last expect.
You could use an await if you plan to do something with the request result outside the chain.
Like  
`const response = await request(app).get().expect().expect()
doSomethingWithResponse(response)
`

Reply:
This part I didn't understand, the test passes with or without `await`. But without `await` the underlying route doesn't get called at all, the test coverage will be lower as well. So looks like it returns the promise immediately and all expect() pass somehow.